book-crawler/app/crawler/kyobo.py

```
@@ -352,9 +432,12 @@ def crawl_kyobo_books(limit: int | None = None, max_pages: int = 50, progress_ca
                 "source_field": "CRAWLING"
             })
         
-        # 요청 간격 조절 (봇 탐지 방지)
-        crawl_delay = float(os.getenv("CRAWL_DELAY", "1"))
-        time.sleep(crawl_delay)
+        # 요청 간격 조절 (봇 탐지 방지, 랜덤 지연)
+        crawl_delay = float(os.getenv("CRAWL_DELAY", "3.0"))  # 기본값 증가
+        human_delay(base=crawl_delay, spread=crawl_delay*0.8)
+        
+        # 장기 휴식 (10-15권마다 60-120초)
+        long_break(idx + 1, interval=random.randint(10, 15))
```

Ruff (0.12.2)

Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)